### PR TITLE
feat: University, Nokia, Genetec, Unity sponsor logos

### LINKED ISSUE #84: Add Genetec, Unity, and Nokia sponsors
See Slack (in #tech_design) for the options of logos Genetec (and Unity?) explicitly sent to us. The others can be a placeholder one from their site, searching for it, or something like https://www.logo.wine/ 

- Genetec and Unity are **gold** sponsors
- Nokia is a **silver** sponsor

### PR DESCRIPTION
**Pull request by gordonpn**
Looking for feedback on the layout of the logos on the mobile width of the sponsor section!

closes #84 #85

### CODE REVIEW

File: components/sections/Sponsors.tsx [hunk 1]
```diff
@@ -27,14 +28,29 @@ const thinGrid = [
...
+  .map((row) => `"${row}"`)
+  .join(" ");
+
+const thinSchoolGrid = [
+  `calgary . carleton`,
+  `toronto . uottawa`,
+  `. waterloo.`,
+]
+  .map((row) => `"${row}"`)
+  .join(" ");
```
Comment (sttic) [inline components/sections/Sponsors.tsx:52]: Good idea making another grid to better fit the 5 universities!

**Review by sttic — COMMENTED** [review]
I think mobile layout looks good! My only nitpick would be to 'normalize' the extra spacing/viewbox in the SVGs since it's hard to accommodate the differences they each have. Given the same width, different images could look smaller because of this (i.e. Waterloo looks a tad smaller):

(note image areas are highlighted in below image)

image

Since they're SVGs, what I normally do is throw them in Figma and use the "Resize to fit" option so the actual content/logo takes up the full viewbox.

**Comment by gordonpn** [comment]
> Since they're SVGs, what I normally do is throw them in Figma and use the "Resize to fit" option so the actual content/logo takes up the full viewbox.

@sttic Thanks for the tip! I've just updated the PR with all the new SVGs resized to fit.

File: components/sections/Sponsors.tsx [hunk 2]
```diff
@@ -119,9 +151,48 @@ export default function Sponsors(): ReactElement {
...
+            <SponsorImage
+              src={`${imgPath}/universities/carleton.svg`}
+              gridArea="carleton"
+              width={["60%", "60%", "25%"]}
+              minWidth="72px"
+            />
+            <SponsorImage
+              src={`${imgPath}/universities/toronto.svg`}
+              gridArea="toronto"
+              width={["60%", "60%", "50%"]}
```
Comment (sttic) [inline components/sections/Sponsors.tsx:180]: Hmm I'm just noticing this now but on the last breakpoint, should this (and waterloo's) be set to "25%" like the rest?

Comment (gordonpn) [reply components/sections/Sponsors.tsx]: Thanks for catching that! I just pushed a fix

**Review by sttic — APPROVED** [review]
Sorry, 1 last change request :sweat_smile: I think there should be under the sponsors (between it and ticket pricing section). I believe the previous "Stay tuned for more!" message was adding some margin but the margin wasn't re-added since its removal.

Besides that it looks good to go! Feel free to merge whenever after this. Thanks for the PR and the updates :grin:

**Comment by gordonpn** [comment]
> Sorry, 1 last change request 😅 I think there should be more space under the sponsors (between it and ticket pricing section). I believe the previous "Stay tuned for more!" message was adding some margin but the margin wasn't re-added since its removal.

That's totally my bad! LOL

**Pull request merged**